docs/source/reference/index.rst

```diff
@@ -1,147 +1,143 @@
-.. module:: pytorch_pfn_extras
```

Comment:
Remove the module directive, as it impacts the links to the pytorch_pfn_extras package